policyengine_us/variables/gov/states/in/tax/income/exemptions/in_base_exemptions.py

```diff
@@ -12,4 +12,14 @@ class in_base_exemptions(Variable):
 
```

Comment:
Please split the formula into these components for clarity. In general we try to apply this in cases where the formula ends in a summation.
```suggestion
    adds = ["in_base_filer_exemptions", "in_base_dependents_exemptions"]
```